### ISSUE TITLE
Plex can't play anything

### ISSUE THREAD

**Issue by taynotfound** [cabernetwork/cabernet#144]
<img width="764" alt="image" src="https://github.com/cabernetwork/cabernet/assets/50525948/9fc9889c-00fb-44de-aa94-fca89e322f63">

Just installed cabernet, and plex on my Server

**Comment by rocky4546** [comment]
Well, since you have not given any information at all, it could be almost anything.  Most probable is that your environment is setup so that Cabernet and Plex cannot talk on your network.  If I was to take a very wild guess, I would say you are using docker containers, which can cause this type of issue, but that is just a guess.

**Comment by taynotfound** [comment]
Heya. Yea I didn't know what to provide.
I am using Cabernet as a Docker and Plex as a APT. The TV Guide works. Just
the streaming itself is broken

**Comment by taynotfound** [comment]
I tried proxying the ports too to have the Cabernet listen on my Machine's
Port. Nothing really worked out. When I modify the channels.m3u and add my
Servers IP instead of the one Cabernet sets in there it works with VLC. I
tried connecting the Cabernet Instance via my Public IP, 0.0.0.0, and the
Internal IP. Nothing worked really. And the XML EPG seems broken too when
trying to get it into Plex via the Public Facing IP. That might be because
of HTTP and HTTPS tho (will be trying with a domain when I get a tip for a
solution on here as the EPG is 2nd Priority as of rn.

**Comment by rocky4546** [comment]
so, since you are using docker, you need to first establish the network and make sure you understand what it is.  To do this, you will need to hit the 6077 port of Cabernet main website using an IP address from outside of the docker container.  If/When you get that to work, you then need to verify the streaming port network is also open.  Change the port from 6077 to 5004.  If it works, you will get the response "501 - Not Implemented" from Cabernet.  Once you have this working, the next thing is to read some instructions from Cabernet on how to implement docker setting changes.  Let me know when you have 5004 working.

**Comment by taynotfound** [comment]
All the ports directly worked out of the Box and are listening on the same
port on my IP. I even put a Domain on it too. So that shouldn't be an issue.

**Comment by taynotfound** [comment]
So yea, i just tried adding the Cabernet via my Domain. Didnt work too. Not when being in the Server itself, and not on my PC that isnt in the same Network. It does work on VLC on my PC if i add the channels.m3u on VLC tho. So something is wrong with Plex as it seems

**Comment by taynotfound** [comment]
<img width="807" alt="image" src="https://github.com/cabernetwork/cabernet/assets/50525948/c448b341-41cb-473b-a6dd-b756e348e15f">
So now i cant even complete the setup with it

**Comment by rocky4546** [comment]
Continuing with the docker setup.  Per the Readme, step 4c in the docker setup, it states to set the plex_docker_ip with the ip you used to connect to the Cabernet website.  To validate this, you need to request a channels.m3u from Cabernet (through the links page) and see if the URLs in the m3u have changed to your setting.

**Comment by taynotfound** [comment]
Yes. Thats what i did. It works completely flawless with VLC. Just plex seems to have the Issues. It is wonky when trying over the app.plex.tv Domain, IP of the Server, Binding the Server's port to localhost. That all doesnt work.

**Comment by taynotfound** [comment]
So i looked even deeper into it. And seemingly only the German Channels (The Country i live in) work on Plex. Even tho they work completely fine on VLC

**Comment by rocky4546** [comment]
Cabernet should now be setup.  Considering you still are not providing setup info, let us continue with the plex setup.  For plex, go to the "Live TV & DVR" area and go to "Don't see your HDHomeRun device? Enter its network address manually".  Then enter the address http://ipaddress:6077/
continue following the instructions and enter the xmltv.xml url when asked.

**Comment by taynotfound** [comment]
Yes. I did that. It is all added Fine. On Plex it does play like 2 Channels as of rn and one of them is choppy (even tho that might either be my wifi or the Channel willingly breaking their stream).

Here is my Config.ini

```ini
[loggers]
keys = root

[logger_root]
level = TRACE
handlers = loghandler

[handlers]
keys = loghandler, filehandler

[formatters]
keys = extend,simple

[handler_filehandler]
enabled = False
level = INFO
formatter = extend
class = lib.common.log_handlers.MPRotatingFileHandler
args = (os.getenv('LOGS_DIR','data/logs')+'/cabernet.log', 'a', 10000000, 10)

[handler_loghandler]
enabled = True
level = WARNING
class = StreamHandler
formatter = extend
args = (sys.stdout,)

[formatter_extend]
format = %(asctime)s-%(levelname)s:%(module)s %(message)s

[formatter_simple]
format = %(levelname)s:%(module)s %(message)s

[main]
uuid = 0ACADBC6-FFCF-11EE-9BAB-0242AC110002
maintenance_mode = False
use_encryption = False

[hdhomerun]
hdhr_id = 1059F5F5

[display]
theme = appletv

[daddylive]
hdhr_id = 105FA4CE
enabled = True

[daddylive_default]
label = DaddyLive Instance
enabled = True

[tvguide]
enabled = True

[m3u]
hdhr_id = 1056F74F
enabled = True

[plutotv]
hdhr_id = 105A8623
enabled = True

[xumo]
hdhr_id = 10582BEB
enabled = True
channellistid = 10006

[xumo_default]
label = XUMO Instance
enabled = True

[plutotv_default]
label = PlutoTV Instance
enabled = True

[epg]
epg_prettyprint = True

[web]
local_ip = 0.0.0.0
plex_accessible_ip = PUBLICIPOFTHESERVER

```
Plex is installed on the Same Server.
My Server is located in a Datacenter in Frankfurt. That's abt all info i can give

**Comment by rocky4546** [comment]
Not sure why you have the local_ip set.  Tool tip says that may be an issue.  I would probably delete the line.

Sounds like everything is working.  Ability to play is based on the quality of the provider which I cannot change.  for daddylive, I check the chat server for status.  Sometimes a channel will be down for a while. I even have asked for a channel to be reset to get it to play well.

**Comment by taynotfound** [comment]
okay, i removed it. but isnt it odd that just 2 channels seem to work?
